### PR TITLE
Blog votes

### PR DESCRIPTION
**Pull request by hung3a8**
# Description

Type of change: New feature

## What

Users can now vote for personal and global blogs. Organization blogs are excluded.

Preview:
image

# How Has This Been Tested?

- [x] Local test

# Checklist

- [x] I have explained the purpose of this PR.
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

By submitting this pull request, I confirm that my contribution is made under the terms of the AGPL-3.0 License.

### CODE REVIEW

File: judge/views/blog.py [hunk 1]
```diff
@@ -1,25 +1,95 @@
...
+                    continue
+                return HttpResponseBadRequest(_('You cannot vote twice.'), content_type='text/plain')
+        else:
+            BlogPost.objects.get(id=blog_id).vote(delta)
+        break
+    return HttpResponse('success', content_type='text/plain')
+
+
+def upvote_blog(request):
+    print('lmao')
```
Comment (leduythuccs) [inline judge/views/blog.py:85]: remove

File: requirements.txt [hunk 2]
```diff
@@ -36,7 +36,7 @@ lupa
 martor @ git+https://github.com/VNOI-Admin/martor.git
 netaddr
 webauthn<1
-bleach
+bleach>=4.1.0,<5.0.0
```
Comment (leduythuccs) [inline requirements.txt:39]: move this change to master.

**Review by leduythuccs — CHANGES_REQUESTED** [review]
seem gud, required some changes

**Review by hieplpvip — COMMENTED** [review]
LGTM

**Review by leduythuccs — APPROVED** [review]
LGTM

**Pull request merged**